plugins/azure-sites/src/components/AzureSitesOverviewTableComponent/AzureSitesOverviewTable.tsx

```diff
@@ -73,9 +73,11 @@ const Kind = ({ value }: { value: Kinds }) => {
     app: <PublicIcon />,
     functionapp: <FlashOnIcon />,
   };
+  const kindArray = value.split(',');
```

Comment:
What is the value that you are getting when you try this? It looks like it should either be `'app' | 'functionapp'` why do you get a comma seperated list? We should update the types if this is not the case too.